app/controllers/staff/sponsors_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Staff::SponsorsController < Staff::ApplicationController
+  before_action :enable_website_subnav
+
+  def index
+    @sponsors = current_event.sponsors
+  end
+
+  def new
+    @sponsor = current_event.sponsors.build
+  end
+
+  def create
+    @sponsor = current_event.sponsors.build(sponsors_params)
+
+    flash[:success] = "Sponsor was successfully created."
+    redirect_to event_staff_sponsors_path if @sponsor.save
+  end
+
+  def edit
+    @sponsor = current_event.sponsors.find_by(id: params[:id])
+  end
+
+  def update
+    @sponsor = current_event.sponsors.find_by(id: params[:id])
+    @sponsor.update(sponsors_params)
+    flash[:success] = "#{@sponsor.name} was successfully updated."
+    redirect_to event_staff_sponsors_path
+  end
+
+  def destroy
+    @sponsor = current_event.sponsors.find(params[:id])
+    @sponsor.destroy
+    redirect_to event_staff_sponsors_path
+    flash[:info] = "Sponsor was successfully removed."
+  end
+
+  private
+  def sponsors_params
```

Comment:
I think the convention would be to name this `sponsor_params` not that it really makes a difference.

Reply:
Good catch!